app/keepers/keepers.go

```diff
@@ -588,6 +588,7 @@ func NewAppKeeper(
 		distrkeeper.NewMsgServerImpl(appKeepers.DistrKeeper),
 		wasmkeeper.NewMsgServerImpl(&appKeepers.WasmKeeper),
 		appKeepers.WasmKeeper,
+		appKeepers.AccountKeeper,
```

Comment:
We are already giving accountKeeper as a parameter.

Reply:
In `RegistPrecompiledContract()`, all keeper arguments are interface
May I use `pwasm.AccountKeeper` in `auth` module?